classes/classes/Appearance.as

```diff
@@ -2228,7 +2147,7 @@
 					[SKIN_TYPE_GOO, "goo"],
 					[SKIN_TYPE_UNDEFINED, "undefined flesh"],
 					[SKIN_TYPE_DRAGON_SCALES, "scales"],
-					[SKIN_TYPE_FISH_SCALES, "scales"],
+					[SKIN_TYPE_FISH_SCALES, "scales"]
```

Comment:
Leaving trailing commas in array definitions is not a mistake, but common practice, so adding more items to the end of it only affects one line. That among other advantages.